src/tests/user/userData.js

```diff
@@ -0,0 +1,13 @@
+const userData = {
```

Comment:
### Nitpick

- You can also tryout `faker.js`

Reply:
I impremented faker